app/views/idv/phone_question/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<% title @title %>
```

Comment:
If the title never changes, could we have just inlined the text to avoid the indirection?

```suggestion
<% title t('doc_auth.headings.phone_question') %>
```